docs/decision_record.md

```diff
@@ -8,6 +8,41 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 [TOC]
 
+## 2023-08-22 - Seqcol schemas MUST specify collated attributes with a local qualifier
+
+### Decision
+
+Collated attributes are seqcol attributes where the values of the attribute are 1-to-1 with sequences in the collection, and represented in the same order as the sequences in the collection. Names, lengths, and sequences are examples of collated attributes. While not strictly required for the core functionality of sequence collections, we anticipate it downstream applications will benefit if the seqcol JSONschema specifies which attributes are collated. We therefore REQUIRE the seqcol JSONschema to specify which attributes are collated, and these will be specified using a local qualifier (defined below). 
```

Comment:
Ok I specified it's a boolean, and added examples of collated and non-collated attributes.

I don't believe we discussed default value. 
- Since we indicate that you *must specify which attributes are collated*, this seems to imply the default is false.
-  But, I guess it could go either way, since the basic names/lengths/sequences are all collated, it might make more sense to make the default true.
- Or we could say you must specify for every attribute.

I guess I don't have strong opinions here.